sire/advanced.md

```diff
@@ -26,70 +26,104 @@ This line is the type signature:
 > Bar > Bar > Nat > Bit
 ```
 
-First parameter is a bar ("needle"), second parameter is a bar ("haystack"), third parameter is a nat (offset) and the function returns a bit (boolean).
+First parameter is a bar (`needle`), second parameter is a bar (`haystack`), third parameter is a nat (`off`, for offset) and the function returns a bit (boolean).
 
 ## Lambdas
 
 - `&`  Anonymous lambda
 - `?`  Named lambda
-- `??` Named and pinned lambda (Pinning has to do with helping with memory layout in persistence. Don't worry about it too much for now, but you'll come across it in source files)
+- `??` Named and pinned lambda (Pinning has to do with memory layout. Don't worry about it too much for now, but you'll come across it in source files)
 
-## Col macro
+For example, here's `&` used infix:
```

Comment:
Very good to introduce the lambda example (in concert with demonstrating prefix + infix) as a tee-up to col explanation.